geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,131 @@
+"""LogNormal Sampler"""
+
+import geomstats.backend as gs
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.spd_matrices import SPDMatrices
+
+
+class LogNormal:
+    """LogNormal Sampler on manifold of SPD Matrices and Euclidean Spaces.
+
+    (1)For Euclidean Spaces, if X is distributed as Normal(mean, cov), then
+    exp(X) is distributed as LogNormal(mean, cov).
+
+    (2)For SPDMatrices, LogNormal distribution is defined by embedding mean
+    in the Euclidean Space; mean(a SPD matrix) is embedded into
+    Symmetric Matrices Space using Matrix Logarithm, which is then embedded
+    into Euclidean Space. This embedded mean and given cov is used to
+    parametrize Normal Distribution.
+
+    Parameters
+    ----------
+    manifold: Manifold obj, {Euclidean(n), SPDMatrices(n)}
+        Manifold to sample over. Manifold should
+        be of instance of Euclidean or SPDMatrices
```

Comment:
NIT: missing period at the end of parameter description

(also at other places below)